From 342c4b588f2f069a2df2157ac166237a54277b3a Mon Sep 17 00:00:00 2001
From: Ian McInerney <ian.s.mcinerney@ieee.org>
Date: Fri, 8 Apr 2022 01:27:52 +0100
Subject: [PATCH] Call the proper wxBitmap constructor for XPM data

wxBitmap back to at least 3.0.0 did not have a constructor for an XPM
image that took a size parameter, it would only take the XPM data
itself.

I don't know how this compiled until now, but it doesn't anymore when
the size is included and built against wx 3.1.6 in Fedora.
---
 src/MixerBoard.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/MixerBoard.cpp b/src/MixerBoard.cpp
index 54b5f146c..7fe816154 100644
--- a/src/MixerBoard.cpp
+++ b/src/MixerBoard.cpp
@@ -1307,7 +1307,7 @@ void MixerBoard::LoadMusicalInstruments()
    wxMemoryDC dc;
 
    for (const auto &data : table) {
-      auto bmp = std::make_unique<wxBitmap>(data.bitmap,24);
+      auto bmp = std::make_unique<wxBitmap>(data.bitmap);
       dc.SelectObject(*bmp);
       AColor::Bevel(dc, false, bev);
       mMusicalInstruments.push_back(std::make_unique<MusicalInstrument>(
-- 
2.34.1

